### PR TITLE
allure map to json

### PR DESCRIPTION
**Pull request by Kristanee**
allure

### CODE REVIEW

**Comment by Kristanee** [comment]
allure

File: from_ge_to_allure_mapper/allure.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import from_ge_to_allure_mapper.map_to_json as allure
+
+# create_report = from_ge_to_allure_mapper.create_json_report()
```
Comment (a-chumagin) [inline from_ge_to_allure_mapper/allure.py:3]: is there reason to keep commented line?

File: from_ge_to_allure_mapper/map_to_json.py [hunk 2]
```diff
@@ -113,7 +119,7 @@ def get_stop_test_time(file):
 
 def get_params(file):
     params = file['expectation_config']['kwargs']
-    del params['result_format']
+    # del params['result_format']
```
Comment (a-chumagin) [inline from_ge_to_allure_mapper/map_to_json.py:122]: delete?

**Comment by Kristanee** [comment]
https://github.com/provectus/from_ge_to_allure_mapper/pull/8/commits/aace158b23b31d60f6681977113313ba275aea83

Pls check changes

**Pull request merged**